clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/ModifyAsgAtomicOperation.groovy

```diff
@@ -78,6 +80,20 @@ class ModifyAsgAtomicOperation implements AtomicOperation<Void> {
           .withHealthCheckType(description.healthCheckType)
           .withTerminationPolicies(description.terminationPolicies)
 
+      if (description.disabledGroupMetrics) {
```

Comment:
@cfieber yea, should be doable. We can check the `EnabledMetrics` field on the `AutoScalingGroup`.